pageserver/client/src/mgmt_api.rs

```diff
@@ -217,6 +217,19 @@ impl Client {
         }
     }
 
+    pub async fn tenant_time_travel_remote_storage(
+        &self,
+        tenant_shard_id: TenantShardId,
+        timestamp: &str,
+        done_if_after: &str,
+    ) -> Result<StatusCode> {
+        let uri = format!(
+            "{}/v1/tenant/{tenant_shard_id}/time_travel_remote_storage?travel_to={timestamp}&done_if_after={done_if_after}",
+            self.mgmt_api_endpoint
+        );
+        Ok(self.request(Method::PUT, &uri, ()).await?.status())
```

Comment:
No need to return the status here: `self.request` already generates an Err result if the status was a failure.  That will let you skip the extra check of status code at the call site as well.

Reply:
(the delete method above returns the statuscode because there are multiple successful cases for deletion: 202, 404, and callers need to discriminate.  But most methods don't need that.)